README.md

```diff
@@ -87,6 +87,7 @@ METAPLEX_AUTH_TOKEN=secret
 # needs to be real so create a personal magic.link account or use the staging key
 MAGIC_SECRET_KEY=secret
 LOGTAIL_TOKEN=secret
+PRIVATE_KEY=xmbtWjE9eYuAxae9G65lQSkw36HV6H+0LSFq2aKqVwY=
```

Comment:
I'm guessing this is some kind of dev/testing key(??), but I think a comment would be good to explain what this specific value is and maybe which public key it's linked to.